doc/TESTING.md

```diff
@@ -22,36 +22,64 @@ If you prefer to install a .deb package directly, you can [download one here](ht
 
 Unlike the repository installation method, you need to download and install Nicotine+ from the link above every time you want to update to the latest unstable build.
 
-### Flatpak
+### Other Distributions
+
+#### Flatpak
 
 Unstable [Flatpak](https://www.flatpak.org/setup/) packages are built after every commit to the master branch.
 
 - [Download Unstable Flatpak Package](https://nightly.link/nicotine-plus/nicotine-plus/workflows/packaging/master/flatpak-package.zip)
 
```

Comment:
_Insert a copy of the sentence here:_

#### "pip"
"If Nicotine+ is not packaged for your system, the latest unstable build can be [installed using pip (see below)](TESTING.md#pip)."

Reply:
@slook I'd do so however creating a "pip" section means the linked text will just link to its own section and not the "pip" later on under "Cross-Platform". I could name the proposed section something else, but what? "Python"? Either that or omit the "pip" header and just add a sentence after the Flatpak text?

Maybe just name the section "Alternatives" and add the text "Nicotine+ can also be installed via cross-platform methods (see below)."?

Reply:
Oh right, of course yes my mistake.

Just put the sentence on it's own without any header then.